public/pwa.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>PWA Test page</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="apple-mobile-web-app-status-bar" content="#aa7700">
+    <meta name="theme-color" content="black">
+    <link rel="apple-touch-icon" href="/images/icon-192x192.png">
+    <link href="manifest.json" rel="manifest">
+  </head>
+  <body>
+    <img src="images/icon-192x192.png"/>
```

Comment:
This is needed for Lighthouse `performance` audit, without it, it will just keep waiting for something to be painted